FutureNHS/futurenhs-platform
# AB#1789 - Show Folders in Workspace (UI)

![](https://media.giphy.com/media/l2Je5Mochl7iyWHoQ/giphy.gif)

[Ticket](https://dev.azure.com/futurenhs/FutureNHS/_workitems/edit/1789)

[Figma](https://www.figma.com/file/tYFsqhG61CleWS1tUapAar/Source-of-Truth-UI?node-id=47%3A0)

[Deployed Here]https://fnhs-dev-david.westeurope.cloudapp.azure.com/workspaces/2843ac55-9325-4785-97be-f8140533fc72/folders/8d45ef39-8a8a-4bed-be37-a2ae5d389c08)

Screenshot:
![Screenshot 2020-09-21 at 17 01 17](https://user-images.githubusercontent.com/254647/93791509-41262180-fc2c-11ea-98bc-d4f4e02d62d3.png)

Folders list closed:
<img width="439" alt="Screenshot 2020-09-21 at 17 20 22" src="https://user-images.githubusercontent.com/13286587/93793501-cd394880-fc2e-11ea-9a82-e99cc00d7dc9.png">

Folder selected in navigation, folder homepage visible:
<img width="1077" alt="Screenshot 2020-09-21 at 17 20 09" src="https://user-images.githubusercontent.com/13286587/93793654-01ad0480-fc2f-11ea-8917-a033410fd883.png">



As a user
I want to see folders in workspaces
So that I can navigate to the file I want
## Acceptance Criteria

## Pre-review checklist:

- [x] Tests (will do Selenium tests in a separate PR)


- [ ] Analytics (will happen in a separate PR)

## Testing information

1) Navigate to a workspace homepage
2) Admire nav bar to the left of screen
3) Note list of folders in the nav bar (use `create-folders`script in `workspace-service/scripts` to create if you don't have any in workspace already)
4) Verify that when you select a folder, the folder icon changes to 'open', your url changes to the 'folder homepage', and that folder is highlighted'

## Test:


- Platforms/Browsers:

  - [ ] Google Chrome (latest version)



- Accessibility:



  - [ ] Magnification


  - [ ] Text size 200%

  - [ ] Touch target areas (Mobile)

  - [ ] Landscape (Mobile)
